src/drawable.h

```diff
@@ -69,7 +54,21 @@ enum Priority {
  */
 class Drawable {
 public:
-	Drawable(DrawableType type, int z, bool is_global);
+	/** Flags with dictate certain attributes of drawables */
+	enum class Flags : uint32_t {
+		/** No flags */
+		None = 0,
+		/** This is a global drawable which will appear in all scenes */
+		Global = 1,
+		/** This is a shared drawable which will appear in all scenes that use shared drawables */
+		Shared = 2,
+		/** This flag indicates the drawable should not be drawn */
+		Invisible = 4,
+		/** The default flag set */
+		Default = None
```

Comment:
Can you document which of these Flags are "(not) affected by transitions"?

Reply:
Not sure what you mean? These flags are all orthogonal to transitions.

Reply:
I mean global drawables are never faded in or out. They stay "above" transitions (or screen show/erase stuff).

Reply:
That's purely due to Z values not flags. Or at least it will be once I do the PR to kill the global list.

Globals are rendered above transitions.

Reply:
ah okay, I was still thinking in the global vs local list scope